bench/bench_lru.c

```diff
@@ -143,7 +143,7 @@ static struct key * read_lines(const char *fname, size_t *count, char **pfree)
 	return result;
 }
 
-// compatibility layer for the oler lru_* names; it's more compler with lru_create
+// compatibility layer for the older lru_* names; it's more complete with lru_create
```

Comment:
I couldn't guess for the second word here

Reply:
I apparently wrote the line, but I'm unable to figure out what I meant, even after understanding (again) what this compatibility layer is about.  I suggest to just remove the part starting at the semicolon.

Anyway, we surely don't even need this compatibility anymore... to micro-benchmark LRU's API that's been gone for over five years.

Reply:
"compatibler" came to my mind, but it would seem really weird.